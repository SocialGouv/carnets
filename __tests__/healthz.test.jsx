import { expect, test } from "vitest";
import { render } from "@testing-library/react";
import Page from "../src/app/healthz/page";

test("Page", () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
