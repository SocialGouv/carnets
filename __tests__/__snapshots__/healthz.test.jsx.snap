// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Page 1`] = `
<div>
  <div>
    It Works!
  </div>
</div>
`;
